src/components/EditOrganization/index.tsx

```diff
@@ -97,9 +118,14 @@ export default function OrganizationView({ organization, reloadOrganizationData
     if (organizationRecord.name === '') {
       setNameError('Required field.');
     } else {
+      if (!tzChangedFromDefault) {
+        organizationRecord.timeZone = defaultTimeZone;
+      }
```

Comment:
actually we may not need to do this..
there's another big ticket to initialize user and org time zones.. https://terraformation.atlassian.net/browse/SW-2708
for now, might be good do assume the org retains it's original time zone (if not set in the selector) - even if it is undefined.